### PR TITLE
Data dictionary edits for usage events/attributes

### PR DESCRIPTION
**Pull request by zuluecho9**
PMMs who were auto-tagged; you can ignore. 

Pricing/usage liaison work, edits to data dictionary: 

* Correct 'full user' language and remove 'Full Stack Observability' mentions, make more clear
* Add new core-version pricing attributes (FullPlatformUsersBillable, and FullPlatformUsersFree)
* Deleted applied intelligence-related attributes

### CODE REVIEW

**Comment by zuluecho9** [comment]
Got review from @kmcfarland

File: src/data-dictionary/events/NrMTDConsumption/NrMTDConsumption.event-definition.md [hunk 1]
```diff
@@ -5,4 +5,4 @@ dataSources:
   - Account Usage
 ---
 
-This event tracks your New Relic platform usage and estimated cost across an entire billing period. It generates aggregate values from the NrConsumption event.
\ No newline at end of file
+This event tracks your New Relic usage (data and billable users) and estimated cost across an entire calendar month. It generates aggregate values from the NrConsumption event.
```
Comment (bradleycamacho) [inline src/data-dictionary/events/NrMTDConsumption/NrMTDConsumption.event-definition.md:8]:
```suggestion
This event tracks your New Relic usage (data and billable users) and estimated cost across an entire calendar month. It generates aggregate values from the `NrConsumption` event.
```

Comment (bradleycamacho) [reply src/data-dictionary/events/NrMTDConsumption/NrMTDConsumption.event-definition.md:8]: Honestly not sure if that has to be in code or not.

**Review by bradleycamacho — APPROVED** [review]

File: src/data-dictionary/events/NrMTDConsumption/NrMTDConsumption.event-definition.md (hunk 1 above)
Comment (zuluecho9) [reply src/data-dictionary/events/NrMTDConsumption/NrMTDConsumption.event-definition.md:8]: you're right that it generally should but I'm actually not sure if the data dictionary event descriptions can show code format but I'll try it.

**Pull request merged**